### ISSUE TITLE
Ranger open same file existing in other windows will overwrite the existing window

### ISSUE THREAD

**Issue by ztlevi** [punassuming/ranger.el#190]
#### Description
If Emacs with split windows，open the same file existing in other windows will overwrite the exsiting window. 

#### Reproduction
- Start Emacs
- split window
![image|591x500](https://emacs-china.org/uploads/default/original/2X/b/b80a4f6d20b920336669a5b94a653e1fae61fa98.png) 

- open the same file using dired or ranger
![image|591x500](https://emacs-china.org/uploads/default/optimized/2X/8/87d017a82ddee88839ded2cb2fe9553481521a60_1_591x500.png) 

- split windows are merged
![image|591x500](https://emacs-china.org/uploads/default/optimized/2X/c/cc69ad2326989fd8e1d9f44c5b6df8d40f7a7e39_1_591x500.png)

#### Observed behaviour
split windows are mereged

#### Expected behaviour
Do not merge split windows

**Comment by punassuming** [comment]
This is an issue only in deer-mode right?

**Comment by ztlevi** [comment]
Yes, because we probly will not open two ranger buffers at the same time

**Comment by ztlevi** [comment]
I do like open two deer buffers at the same time. For example, I can do copy pasting files between these two deer buffers. 

I've noticed deer can create multiple tabs. But I prefer open multiple deer buffers.

**Comment by thanhvg** [comment]
I have another issue with current behavior of `deer-mode` when opening the same file. After the split window is deleted because ranger is opening the same file, do a `deer` on the buffer to go to `deer-mode` will trigger a warning: 
```
file changed on disk; really edit the buffer? (y, n, r or C-h)
```
The buffer is not changed at all. I guess `ranger` misread the buffer state in this case.
And there is no way to get out of this. I find @ztlevi's solution fixed my problem. 
I am using `Spacemacs` so it could be `Spacemacs`'s issue, but I doubt it.

Thanks
